x-pack/plugins/ml/public/file_datavisualizer/components/fields_stats/fields_stats.js

```diff
@@ -83,8 +83,15 @@ function createFields(results) {
 
         return field;
       } else {
+        // field is not in the field stats
+        // this could be a text field or a field which the endpoint has
```

Comment:
It should be "this could be the `message` field for a semi-structured log file or a field which the endpoint has"